src/types.rs

```diff
@@ -136,25 +136,7 @@ pub enum BinOper {
     As,
     Escape,
     #[cfg(feature = "backend-postgres")]
-    Matches,
-    #[cfg(feature = "backend-postgres")]
-    Contains,
-    #[cfg(feature = "backend-postgres")]
-    Contained,
-    #[cfg(feature = "backend-postgres")]
-    Concatenate,
-    #[cfg(feature = "backend-postgres")]
-    Similarity,
-    #[cfg(feature = "backend-postgres")]
-    WordSimilarity,
-    #[cfg(feature = "backend-postgres")]
-    StrictWordSimilarity,
-    #[cfg(feature = "backend-postgres")]
-    SimilarityDistance,
-    #[cfg(feature = "backend-postgres")]
-    WordSimilarityDistance,
-    #[cfg(feature = "backend-postgres")]
-    StrictWordSimilarityDistance,
+    PgOperators(PgBinOper),
```

Comment:
a minor remark but still: it might be better to have `PgOperator` (or `PgOp`/`PgOper`) as singular rather than plural (suggestion: remove trailing 's')

Reply:
Thank you! I opened new PR: https://github.com/SeaQL/sea-query/pull/507 with fix)